plugins/wrapper/README.md

```diff
@@ -78,15 +78,69 @@ very beginning:
 module Lib = LibAsserts
 ```
 
+## Advanced specification
 
-## Supported Gospel
+The wrapper plugin also supports advanced Gospel features, such as
+logical models and `old` statements,
+allowing more precise and expressive specifications for abstract types.
+Let's consider a toy example of a polymorphic container type with limited capacity:
 
-The wrapper plugin has currently some limitations on what Gospel
-specifications are supported. Apart from the general restriction to
-the executable fragment of Gospel (as mentioned in the [main README]),
```

Comment:
We also need to keep the information about the general restrictions. You have lifted the imitations coming for the wrapper part of Ortac/Wrapper, but we still have the limitations coming from ortac-core.